### ISSUE TITLE
Using a@href for subpages can scrape content only for one selector

### ISSUE THREAD

**Issue by kvetoslavnovak** [lapwinglabs/x-ray#81]
Hello, I am trying to follow links and get content from sub pages. 
Big help is this working example from stackoverflow: http://stackoverflow.com/questions/31802903/nodejs-x-ray-web-scraper-how-to-follow-links-and-get-content-from-sub-page

Working pattern is
```
      var Xray = require('x-ray');
      var x = Xray();
      x('http://stackoverflow.com/questions', '#questions .question-summary .summary', [{
              title: 'h3',
              link: 'h3 a@href',
              details: x('h3 a@href', {
                  title: 'h1',
                  question: '.question .post-text',
              })
          }])
          (function(err, obj) {
              console.log(err);
              console.log(obj);
          }) 
```
I am trying to use this on another page with similar structure. But on subpages the are more than one titles as well as post text. 

To grab just mulitple titles it works ok. Tu grab just multiple post texts it work as well. But to grab multiple titles as well as post texts it grabs just empty array. 

In other words [ .a ] and [ .b ] works for subpages ok but  [{ .a, .b }]  does not work correctly.

**Comment by kvetoslavnovak** [comment]
Closing as duplicate of https://github.com/lapwinglabs/x-ray/issues/65
